spdb/project/resource.py

```diff
@@ -266,6 +269,15 @@ def is_image(self):
             return True
         else:
             return False
+    
+    def is_cloudvolume(self):
+        """
+        Check if channel is a cloudvolume layer.
+
+        Returns:
+            (bool): True if channel is cloudvolume. 
+        """
+        return self.storage_type == "cloudvol" and len(self.cv_path) > 0
```

Comment:
Another example why we need to rip all those constants in the Django models out into their own repo so they can be shared by all their users. 😢